testutils/src/main/scala/za/co/absa/enceladus/testutils/e2eSparkRunner/E2ESparkRunner.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.testutils.e2eSparkRunner
+
+import org.apache.log4j.{LogManager, Logger}
+import za.co.absa.enceladus.testutils.HelperFunctions
+import za.co.absa.enceladus.testutils.models.ProcessMeasurement
+
+import sys.process._
+import scala.io.Source
+
+object E2ESparkRunner {
+  private val log: Logger = LogManager.getLogger(this.getClass)
+
+  private val stdJarPath: String = "$DCE_JAR_PATH/$DCE_STD_SPT_JAR"
+  private val confJarPath: String = "$DCE_JAR_PATH/$DCE_CNFRM_SPT_JAR"
+  private val testUtilsJarPath: String = "$DCE_JAR_PATH/$TEST_UTILS_JAR"
+
+  private val stdClass = "--class za.co.absa.enceladus.standardization.StandardizationJob"
+  private val confClass = "--class za.co.absa.enceladus.conformance.DynamicConformanceJob"
+  private val compClass = "--class za.co.absa.enceladus.testutils.datasetComparison.ComparisonJob"
+
+  private def getComparisonConf(newPath: String, refPath: String, outPath: String, keys: String): String = {
+    s"--raw-format parquet --new-path $newPath --ref-path $refPath --out-path $outPath --keys $keys"
+  }
+
+  private def getStandartizationPath(cmd: CmdConfig): String = {
```

Comment:
I would move these to `CmdConfig` as they seem very closely related to its internal state. I would have `CmdConfig` have a command line output function that translates the validated input args into the necessary output args for the specific spark jobs that need to run. This way we encapsulate and limit changes to one class when we start refactoring the runtime options we need for `Standardization` and `Conformance`.

Also, this way we can also avoid `String::split` and accessing arguments based via index. The options can be extracted as constants (e.g. `dataset-name`) inside the scopt definition and these functions.

Reply:
Moved to CmdConfig